beego.go

```diff
@@ -69,10 +69,13 @@ func Run(params ...string) {
 
 func initBeforeHTTPRun() {
 	// if AppConfigPath is setted or conf/app.conf exist
+	/*
 	err := ParseConfig()
 	if err != nil {
 		panic(err)
 	}
+	*/
```

Comment:
you can't do this as maybe user change the `AppConfigPath` in main function. So we need to run parse the config.